tests/datasets/api_tests.py

```diff
@@ -23,6 +23,7 @@
 import yaml
 from sqlalchemy.sql import func
 
+import tests.test_app
```

Comment:
should this be included here? I don't see any code that references it

Reply:
Without this inclusion the test file fails when run in isolation.